Makefile

```diff
@@ -29,7 +29,7 @@
 
 all: test
 
-include cocotb/share/makefiles/Makefile.inc
+include $(shell cocotb-config --makefiles)/Makefile.inc
```

Comment:
This is unrelated cleanup, right?

Reply:
It is needed to get tests to work inside tox virtual environment.